druid/src/core.rs

```diff
@@ -104,6 +104,12 @@ pub(crate) struct WidgetState {
     pub(crate) viewport_offset: Vec2,
 
     // TODO: consider using bitflags for the booleans.
+    pub(crate) set_enabled: bool,
```

Comment:
I think everything connected to enabled/disabled should either use "enabled" or "disabled" consistently. My initial thought was that `LifeCycle::DisabledChanged(false)` would sound strange, but your probably right, the user will search for disabled, since all widgets are enabled by default.

Reply:
I agree on both points, and also thought `DisabledChanged` sounds a bit weird but I don't mind it. We could also consider being verbose and adding `WasEnabled` and `WasDisabled` as separate events, although this would be a departure from current practice, although it might also be easier to reason about?

Reply:
After thinking about this again `DisabledChanged()` is probably still the best option.